internal/schema/1.6/terraform.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func patchTerraformBlockSchema(bs *schema.BlockSchema) *schema.BlockSchema {
+	bs.Body.Blocks["cloud"].Body.Blocks["workspaces"].Body = &schema.BodySchema{
+		Attributes: map[string]*schema.AttributeSchema{
+			"name": {
+				Constraint: schema.LiteralType{Type: cty.String},
+				IsOptional: true,
+				Description: lang.Markdown("The name of a single Terraform Cloud workspace " +
+					"to be used with this configuration When configured only the specified workspace " +
```

Comment:
```suggestion
					"to be used with this configuration. When configured only the specified workspace " +
```

I haven't checked if this also affects the schemas in past versions, if it does then we should fix that as well IMO.